Meta/Lagom/Tools/ELFBaker/main.cpp

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/Error.h>
+#include <AK/String.h>
+#include <AK/StringView.h>
+#include <AK/Vector.h>
+#include <LibC/elf.h>
+#include <LibCore/File.h>
+#include <LibELF/Image.h>
+#include <LibMain/Main.h>
+
+using ELF32BitsImage = ELF::BitnessSpecificImage<Elf32_Ehdr, Elf32_Phdr, Elf32_Shdr>;
+using ELF64BitsImage = ELF::BitnessSpecificImage<Elf64_Ehdr, Elf64_Phdr, Elf64_Shdr>;
+
+template<typename T>
+class ELFBaker {
```

Comment:
(‘bakerʼ is out of fantasy setting! 😂)

(but in fact it should be something about drow, i guess — it takes an elf, but yields a drow, which is somewhat more important [i.e. people say they bake cookies… 😉].)

Reply:
I think baker is an OK name for now :)